extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -751,21 +756,72 @@ public Builder addColumnCodec(final String columnName, final String codecName, f
          * @param useDictionary The hint value
          */
         public Builder useDictionary(final String columnName, final boolean useDictionary) {
-            final ColumnInstructions ci = getColumnInstructions(columnName);
+            final ColumnInstructions ci = getOrCreateColumnInstructions(columnName);
             ci.useDictionary(useDictionary);
             return this;
         }
 
-        private ColumnInstructions getColumnInstructions(final String columnName) {
-            final ColumnInstructions ci;
+        // /**
+        // * For reading, provides a mapping between a Deephaven column name and a parquet column by field id. This
```

Comment:
Is this comment supposed to be a javadoc?

Reply:
It was older javadoc that was relevant to reading. Will remove.